code/core/package.json

```diff
@@ -642,6 +642,7 @@
     "better-opn": "^3.0.2",
     "esbuild": "^0.18.0 || ^0.19.0 || ^0.20.0 || ^0.21.0 || ^0.22.0 || ^0.23.0 || ^0.24.0 || ^0.25.0",
     "esbuild-register": "^3.5.0",
+    "p-limit": "^6.2.0",
```

Comment:
should be a `devDependency`, as it can be prebundled.

Reply:
done